### PR TITLE
WIP: Deployment script draft

### PR DESCRIPTION
**Pull request by arkilic**
@hhslepicka

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+apt-get install sysv-rc-softioc
+if [ ! -d /opt/conda_envs/services/ ]; then conda create -p /opt/conda_envs/services python=3.5; fi
+source activate services
+conda install pymongo ujson tornado jsonschema yaml pytz doct
+pip install requests
+mkdir /services
+cd /services
+mkdir -p /epics/iocs/
+
+t clone https://github.com/NSLS-II/amostra.git
```
Comment (hhslepicka) [inline deploy.sh]: There is this additional "t" in here.

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/bash
+apt-get install sysv-rc-softioc
+if [ ! -d /opt/conda_envs/services/ ]; then conda create -p /opt/conda_envs/services python=3.5; fi
+source activate services
+conda install pymongo ujson tornado jsonschema yaml pytz doct
+pip install requests
```
Comment (hhslepicka) [inline deploy.sh]: Please change it to:
```
pip install requests --upgrade
```

I already changed on the google docs.

File: deploy.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+echo "NAME=amostra
+PORT=$(manage-iocs nextport)
+HOST=$HOSTNAME
+USER=tornado" > config
+
+echo '#!/bin/bash
+export PATH=/opt/conda/bin:$PATH
+source activate services
+python /services/amostra/startup.py --mongo-host=localhost --mongo-port=27017 --database=amostra --service-port=7770 --timezone=US/Eastern' > launch.sh
+
```
Comment (hhslepicka) [inline deploy.sh]: Please add here the following:
```
chmod +x launch.sh
```

I already added at the google docs.

File: deploy.sh (hunk 1 above)
Comment (klauer) [inline deploy.sh]: Or perhaps missing a 'gi'

**Comment by hhslepicka** [comment]
Looks good! Just please implement the changes that I requested and I think that it is good to go.
I needed this changes at LIX when I deployed it.

Do you think that we need to consider the proxy too?
We will need it for the pip install.

File: deploy.sh (hunk 1 above)
Comment (hhslepicka) [inline deploy.sh]: I'm missing a few neurons! 😆

Comment (arkilic) [inline deploy.sh]: We could pull amostra from conda with all its dependencies instead of doing this

**Comment by arkilic** [comment]
Superceded by #32

**Pull request closed without merging**